tests/unit/common/core/test_utils.py

```diff
@@ -25,6 +26,27 @@ def clear_lru_caches() -> Generator[None, None, None]:
     is_saas.cache_clear()
 
 
+def test__is_oss_for_enterprise_returns_false(fs: FakeFilesystem) -> None:
+    # Given
+    fs.create_file("./ENTERPRISE_VERSION")
+
+    # Then
+    assert is_oss() is False
+
+
+def test__is_oss_for_saas_returns_false(fs: FakeFilesystem) -> None:
+    # Given
+    fs.create_file("./SAAS_DEPLOYMENT")
+
+    # Then
+    assert is_oss() is False
+
+
+def test__is_oss_for_oss_returns_true(fs: FakeFilesystem) -> None:
```

Comment:
Remove the unused fixture argument:

```suggestion
def test__is_oss_for_oss_returns_true() -> None:
```

Reply:
Ah, no — we still need the fake file system to ensure the test is isolated. For example, I have an `ENTERPRISE_VERSION` file locally, and if I run this test without the fake file system, it will fail.

Reply:
Ah cool ok!